### PR TITLE
napi: fix property string length

### PR DESCRIPTION
**Pull request by obastemur**

### CODE REVIEW

**Review by MSLaguana — APPROVED** [review]

File: src/node_api_jsrt.cc [hunk 1]
```diff
@@ -783,10 +783,10 @@ napi_status napi_define_class(napi_env env,
 
   JsPropertyIdRef pid = nullptr;
   JsValueRef prototype = nullptr;
-  CHECK_JSRT(JsCreatePropertyId("prototype", 10, &pid));
+  CHECK_JSRT(JsCreatePropertyId("prototype", 9, &pid));
```
Comment (bnoordhuis) [inline src/node_api_jsrt.cc]: Just a suggestion but you could use `sizeof("prototype") - 1` as it's obviously correct and idiomatic.  It's the style the code base uses elsewhere.

Comment (obastemur) [reply src/node_api_jsrt.cc]: @bnoordhuis sounds good.

**Comment by obastemur** [comment]
Went a step further and created a macro.. well, in case we write `"prototype", sizeof("proprotype") - 1` ..

**Comment by MSLaguana** [comment]
Merged in https://github.com/nodejs/node-chakracore/commit/233d9ddd4a15c0e3e7c8fcf05de9de2028c08b90

**Pull request closed without merging**